utils/CHANGELOG.md

```diff
@@ -3,9 +3,13 @@ All notable changes to this module will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 This change log adheres to standards from [Keep a CHANGELOG](http://keepachangelog.com).
 
-## v2 - 2016-11-07
+## v2.1.0 - 2017-05-20
```

Comment:
The version number and date aren't decided yet; If you want to call this section "Unreleased" that'd be helpful :-)